### ISSUE TITLE
ALL MY PROGRESS IS GONE AFTER SAVING!!!

### ISSUE THREAD

**Issue by MEneedhelp101** [photopea/photopea#6651]
Your Operating System :WINDOWS
Your Web Browser: CHROME
after I saved my file my progress on that file was gone. then i tried closing and opening the file again and its like it went backwards meaning it went back to square one when i first started the project.
If there is anyway you can return this file to my account i would be grateful i worked almost 5 hours on this

**Comment by photopea** [comment]
Did you save a file as a PSD? Have you ever been able to save your work as a PSD file to your computer using Photopea? Maybe you have installed some browser extensions into your browser, which prevent Photopea from saving PSD files.

**Comment by Martin-info2** [comment]
It happens to me too; I've encountered this issue when saving files and also when adjusting image DPI. For example, I experienced it when increasing the DPI from 72 to 1200.

**Comment by Martin-info2** [comment]
adding screenshots from the issue:
![After DPI change](https://github.com/photopea/photopea/assets/167081489/b1650749-dca6-4623-a0f7-cc8b2ab51613)
![Before DPI change](https://github.com/photopea/photopea/assets/167081489/32ad3196-c142-4701-a2e0-e3e8c5d9b440)

**Comment by Martin-info2** [comment]
Thank you for responding so quickly.
I'm working on a Mac Air with 4 GB of RAM, so it's still possible that it's a RAM issue. I've tried setting the DPI to 600 and it does work. But previously, less than a week ago, I didn't have this problem. The reason I increased the DPI is that when I zoom in on the resulting PNG, the image is sharper. With lower resolutions, it looks somewhat blurry.
![Screen Shot 2024-04-20 at 20 10 11](https://github.com/photopea/photopea/assets/167081489/ffe46c14-d6d0-4324-a193-ab93a89992fb)
![Screen Shot 2024-04-20 at 20 09 39](https://github.com/photopea/photopea/assets/167081489/da853df9-f0aa-4d62-aa6e-189e166f4a45)
